neondatabase/website
# fix: partners logos glow on Safari

This pull request will fix the animation freezes and glowing logos on the partners page in Safari.

How to review:
- open [Parters page preview](https://neon-next-git-partners-logos-neondatabase.vercel.app/partners) in Safari on Mac, iPad and iPhone
- check that partners logos animation and glowing works correctly without any lags
